geomstats/learning/frechet_mean.py

```diff
@@ -167,7 +167,8 @@ def _ball_gradient_descent(points, metric, weights=None, max_iter=32,
 
         iteration = 0
         convergence = math.inf
-        barycenter = points.mean(0, keepdims=True) * 0
+        barycenter = gs.mean(points, axis=0, keepdims=True)
```

Comment:
How does this work in pytorch, since `axis` is not a parameter of `torch.mean`https://pytorch.org/docs/1.4.0/torch.html?highlight=torch%20mean#torch.mean?